test/integration/extensions_api_deletion_test.go

```diff
@@ -94,12 +95,12 @@ func TestExtensionsAPIDeletion(t *testing.T) {
 		t.Fatalf("unexpected error while waiting for project to delete: %v", err)
 	}
 
-	if _, err := clusterAdminKubeClient.Extensions().HorizontalPodAutoscalers(projName).Get(hpa.Name); err == nil {
+	if _, err := clusterAdminKubeClient.Autoscaling().HorizontalPodAutoscalers(projName).Get(hpa.Name); err == nil {
 		t.Fatalf("HPA object was still present after project was deleted!")
 	} else if !errors.IsNotFound(err) {
 		t.Fatalf("Error trying to get deleted HPA object (not a not-found error): %v", err)
 	}
-	if _, err := clusterAdminKubeClient.Extensions().Jobs(projName).Get(job.Name); err == nil {
```

Comment:
why were these changes needed? I would expect `.Extensions().Jobs` to keep working